app/main.py

```diff
@@ -156,12 +158,19 @@ def upload():
             traceback.print_exc()
 
         if True:
-            flash(filename.replace('.', '-'), 'success-pdf')
-            os.rename(os.path.join(BADGES_FOLDER + "/" + filename + ".badges.pdf"),
-                      os.path.join(BADGES_FOLDER + "/" + filename.replace('.', '-') + "-badges.pdf"))
+            uri = filename.replace('.', '-') + "-badges.pdf"
+            return output('success', 'pdf generation completed successfully', uri)
```

Comment:
@abishekvashok  I think there's a little typo here `url` should be there in place of `uri` in both lines or have you added it but as you are providing link to download , url should be better .
Else looks fine to me, well done 💯

Reply:
Rest API specifies to use uri

Reply:
However I could change that to url @ParthS007

Reply:
Ok , no problem , Let's wait for others to review 👍

Reply:
@ParthS007 I changed it now. :)